### ISSUE TITLE
proc_open() calls blocked on many hosts for security reasons

### ISSUE THREAD

**Issue by bdgeorge** [fontis/fontis_australia#12]
This is in the feed generators, in older versions exec() was used, but both function calls are widely blocked on shared hosting accounts.

The line in question in the current version is:
https://github.com/fontis/fontis_australia/blob/master/app/code/community/Fontis/Australia/Model/FeedCronBase.php#L150

I've not yet gone through the whole thing to see what alternatives there may be, but as is the feeds don't work on hosts that have blocked these.

**Comment by chnorton** [comment]
The feeds generator code has been removed from this extension and moved into its own standalone extension, which can be found here:

https://github.com/fontis/fontis_feedsgenerator

If the problem still still exists, we'd appreciate it if you could please submit as a new issue in that repository.
